src/event-stream/event-stream.service.ts

```diff
@@ -16,7 +16,10 @@
 
 import { HttpService } from '@nestjs/axios';
 import { Injectable, Logger } from '@nestjs/common';
+import { AxiosRequestConfig } from '@nestjs/terminus/dist/health-indicator/http/axios.interfaces';
```

Comment:
Is this the correct import? Feels like maybe this is also exported from something simpler like `@nestjs/axios`

Reply:
That was probably my IDE being "helpful"